terraform/terraform_datadog_monitors.tf

```diff
@@ -0,0 +1,172 @@
+terraform {
```

Comment:
nit :Can remove `terraform_` from file name prefix since it's already in the terraform folder (and it's a `.tf` file).